spec/factories.rb

```diff
@@ -34,4 +34,22 @@
     code { "101" }
     district { "district" }
   end
+
+  factory :ward, class: "Ward" do
+    name { generate(:name) }
+    number { Faker::Number.number(digits: 2) }
+    lsg_body_id { LsgBody.last.id }
+  end
+
+  factory :facility, class: "Facility" do
+    kind { "PHC" }
+    name { Faker::Name.first_name }
+    state { "Kerala" }
+    district { "Ernakulum" }
+    lsg_body_id { LsgBody.last.id }
+    ward_id { Ward.last.id }
+    address { "Test Address Body" }
+    pincode { Faker::Number.number(digits: 7) }
+    phone { Faker::Number.number(digits: 10) }
```

Comment:
We should use [factory bot gem](https://github.com/thoughtbot/factory_bot) and create different files for each model